<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDXSampleClass</class>
 <widget class="QMainWindow" name="QtDXSampleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDXSample</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlayPause"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="paramWidget">
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>111</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="paramWidgetContents">
    <layout class="QHBoxLayout" name="parameterWidgetLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>418</width>
          <height>577</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="parameterScrollLayout"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionPlayPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <activeoff>:/Resources/media-play-16.png</activeoff>
     <activeon>:/Resources/media-pause-16.png</activeon>
     <selectedoff>:/Resources/media-pause-16.png</selectedoff>
     <selectedon>:/Resources/media-play-16.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>PlayPause</string>
   </property>
   <property name="toolTip">
    <string>Start/Pause Animation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtdxsample.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPlayPause</sender>
   <signal>triggered(bool)</signal>
   <receiver>QtDXSampleClass</receiver>
   <slot>toggleAnimation(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>repaintCanvas()</slot>
  <slot>toggleAnimation(bool)</slot>
 </slots>
</ui>
